src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -32974,9 +32975,10 @@ public class C
               InConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
               OutConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
 ");
-
-            Assert.Equal(OperationKind.None, model.GetOperation(node2).Parent.Kind); // PROTOTYPE(ExpressionVariables)
-            Assert.Same(model.GetOperation(node1).Parent.Parent, model.GetOperation(node2).Parent);
+            IOperation operation2 = model.GetOperation(node2);
```

Comment:
Could we get some better names and comments for what exactly these asserts are saying? Just looking at the Assert.Same, I have no idea what the thing being asserted is.